src/compiler/checker.ts

```diff
@@ -3623,7 +3625,8 @@ namespace ts {
                             // Private/protected properties/methods are not visible
                             return false;
                         }
-                    // Public properties/methods are visible if its parents are visible, so const it fall into next case statement
+                        // Public properties/methods are visible if its parents are visible, so

```

Comment:
"falls through" needs to be on its own line for tslint to see it. So the continuation of this comment is on the next line. Added a colon to clarify.